### ISSUE TITLE
Pimcore 10 support

### ISSUE THREAD

**Issue by lpheller** [nambu-ch/pimcore-relation-select-fields#1]
Hi guys,

I was wondering if you plan to update this extension to support Pimcore X / version 10. 
I tried to install the extension but encountered the following error:
```
php bin/console pimcore:bundle:enable RelationSelectFieldsBundle
                                                                                                              
 [OK] Bundle "RelationSelectFieldsBundle\RelationSelectFieldsBundle" was successfully enabled                 

Running post state change commands
----------------------------------

 // Running bin/console assets:install...        

ArgumentCountError {#2059
  #message: "Too few arguments to function Symfony\Component\Config\Definition\Builder\TreeBuilder::__construct(), 0 passed in /Users/lukas/Git/pimcoretest/vendor/nambu-ch/pimcore-relation-select-fields/src/RelationSelectFieldsBundle/DependencyInjection/Configuration.php on line 16 and at least 1 expected"
  #code: 0
  #file: "./vendor/symfony/symfony/src/Symfony/Component/Config/Definition/Builder/TreeBuilder.php"
  #line: 26
  trace: {
    ./vendor/symfony/symfony/src/Symfony/Component/Config/Definition/Builder/TreeBuilder.php:26 { …}
    ./vendor/nambu-ch/pimcore-relation-select-fields/src/RelationSelectFieldsBundle/DependencyInjection/Configuration.php:16 { …}
    ./vendor/symfony/symfony/src/Symfony/Component/Config/Definition/Processor.php:50 { …}
    ./vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Extension/Extension.php:111 { …}
    ./vendor/nambu-ch/pimcore-relation-select-fields/src/RelationSelectFieldsBundle/DependencyInjection/RelationSelectFieldsExtension.php:19 { …}
    ./vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Compiler/MergeExtensionConfigurationPass.php:76 { …}
    ./vendor/symfony/symfony/src/Symfony/Component/HttpKernel/DependencyInjection/MergeExtensionConfigurationPass.php:39 { …}
    ./vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Compiler/Compiler.php:91 { …}
    ./vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/ContainerBuilder.php:749 { …}
    ./vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php:545 { …}
    ./vendor/pimcore/pimcore/lib/Kernel.php:223 { …}
    ./vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php:786 { …}
    ./vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php:125 { …}
    ./vendor/pimcore/pimcore/lib/Kernel.php:202 { …}
    ./vendor/pimcore/pimcore/lib/Bootstrap.php:256 { …}
    ./vendor/pimcore/pimcore/lib/Bootstrap.php:60 { …}
    ./bin/console:44 {
      › /** @var \Pimcore\Kernel $kernel */
      › $kernel = \Pimcore\Bootstrap::startupCli();
      › $application = new \Pimcore\Console\Application($kernel);
    }
  }
}
```
But I was able to get it to work (apparently) by modifying the RelationSelectFieldsBundle\DependencyInjection\Configuration a tiny bit:

```php
    public function getConfigTreeBuilder() {
        $treeBuilder = new TreeBuilder('relation_select_fields');
        $rootNode = $treeBuilder->getRootNode();
   }
```

I'm not that familiar with developing extensions in Pimcore yet  and don't know if this is the right way to go, nor do I know what other changes would be needed to fully support the new version. But since everything seems to work with this change already, maybe it's not that much?

**Comment by dominikschnieper** [comment]
Hi Lukas

We plan to get this extension working with Pimcore 10 within the next few weeks. Thanks for your hint, im glad if it works for you already, so maybe this is really all to change.

**Comment by dominikschnieper** [comment]
Fixed in Version 2.0
